TEAMS_ROLES_STRUCTURES.md

```diff
@@ -383,16 +386,25 @@ User research, collaborations, and products that enable adding IPFS support to p
 - Research and prototype adding IPFS support to package manager clients and mirroring registries onto IPFS
 
 
-### Research Groups
+### Special Interest Groups
 
-The list of existing research groups is:
+The list of existing special interest groups is:
 - https://github.com/ipfs/research-CRDT
 - https://github.com/ipfs/research-blockchain-data
 - https://github.com/ipfs/research-bitswap
 - https://github.com/ipfs/research-p2p-video
 - https://github.com/libp2p/research-pubsub
 - https://github.com/libp2p/research-dht
```

Comment:
Should we rename the repos to be `sig-<name>`?

Reply:
not necessary if it'd break links - but I do think it's a better fit for some of these groups (ex blockchain data)

Reply:
It won't break links, Github does redirects pretty well

Reply:
Cool - I updated the blockchain data one since it seemed more like a sig - but I think the others are explicitly about research and knowledge collection, and therefore don't require name changes unless deemed useful for some reason.